### PR TITLE
Fix ctex to correctly use default Pandoc template

### LINKED ISSUE #322: Couldn't find template file ctex/resources/template.tex
create new rmarkdown from rticles,Couldn't find template file ctex/resources/template.tex,is there any sources where i can download and use it?

---

By filing an issue to this repo, I promise that

- [x] I have fully read the issue guide at https://yihui.name/issue/.
- [ ] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('rticles')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/rticles')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [x] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

### PR DESCRIPTION
**Pull request by cderv**
This will fix #322 

Currently, `ctex()` format is broken because it should use the pandoc default template as stated in 23ecb218be632001c25ae56c6e7d2d30ec14c195 but the changes was not complete. 

We missed that as it was not properly tested. 

Dependency on Rmarkdown needs to be bumped to 1.18, that is the minimum version that use the default pandoc template. 

Also small typo in the skeleton as a duplicated documentclass field was introduced in header.

### CODE REVIEW

File: inst/rmarkdown/templates/ctex/skeleton/skeleton.Rmd [hunk 1]
```diff
@@ -8,7 +8,6 @@ documentclass: ctexart
 keywords:
   - 中文
   - R Markdown
-documentclass: ctexart
```
Comment (yihui) [inline inst/rmarkdown/templates/ctex/skeleton/skeleton.Rmd:11]: This line shouldn't be deleted. It's a document class based on the LaTeX package **ctex**.

File: R/article.R [hunk 2]
```diff
@@ -144,7 +144,9 @@ biometrics_article <- function(..., keep_tex = TRUE, citation_package = 'natbib'
 #' @export
 #' @rdname article
 ctex_article <- function(..., latex_engine = 'xelatex') {
```
Comment (yihui) [inline R/article.R:146]:
```suggestion
ctex_article <- function(..., template = 'default', latex_engine = 'xelatex') {
```

File: R/article.R [hunk 3]
```diff
@@ -144,7 +144,9 @@ biometrics_article <- function(..., keep_tex = TRUE, citation_package = 'natbib'
...
-  pdf_document_format('ctex', latex_engine = latex_engine, ...)
+  pdf_document_format(
+    'ctex', latex_engine = latex_engine, template = "default", ...
+  )
```
Comment (yihui) [inline R/article.R:149]:
```suggestion
  pdf_document_format(
    'ctex', latex_engine = latex_engine, template = template, ...
  )
```

File: inst/rmarkdown/templates/ctex/skeleton/skeleton.Rmd (hunk 1 above)
Comment (cderv) [reply inst/rmarkdown/templates/ctex/skeleton/skeleton.Rmd:11]: But it is already line 7 of the same file ? 
https://github.com/rstudio/rticles/pull/323/files#diff-df04ce7c0c57ebe73c4c44be67b01f45L7

So I get a duplicated field warning from yaml reading.

Comment (yihui) [reply inst/rmarkdown/templates/ctex/skeleton/skeleton.Rmd:11]: Oops. I didn't see that. You are right. We should delete the duplicate.

**Comment by cderv** [comment]
Changes made.

**Pull request merged**